coderd/wsbuilder/wsbuilder.go

```diff
@@ -638,11 +646,19 @@ func (b *Builder) authorize(authFunc func(action rbac.Action, object rbac.Object
 		}
 	}
 
-	if b.logLevel != "" && !authFunc(rbac.ActionUpdate, template) {
+	if b.logLevel != "" && !authFunc(rbac.ActionRead, rbac.ResourceDeploymentValues) {
```

Comment:
Is this the de-facto way to check for admin? It feels like it'd be one of those things that might change in the future (esp. a read permission). Probably fine for now though.

Reply:
Yes, that's why I covered it on the workspace build level with tests for all crucial roles.